status-im/status-legacy
# Update invalid seed phrase copy

### Summary

English copy for Invalid seed phrase warning that pops when trying to recover a multiaccount with a seed phrase that doesn't conform to BIP39 is inaccurate.  This updates the copy to reflect the actual issue.

#### Platforms

- Android
- iOS

##### Functional

- account recovery

### Steps to test

- Open Status
- Select recover account from seed phrase
- Enter invalid seed phrase and try to proceed
- Should get below error message

Error pop-up should say:
`Invalid seed phrase` 
and message should be
`This seed phrase doesn't match our supported dictionary. Some words may be misspelled.`

status: ready

cc @rachelhamlin
